packages/sources/layer2-sequencer-health/src/endpoint/health.ts

```diff
@@ -121,7 +121,11 @@ export const execute: ExecuteWithConfig<ExtendedConfig> = async (request, _, con
   // #3 Option: Check L1 Rollup Contract
   // If every method succeeds, the Network is considered healthy
   // If any method fails, an empty tx is sent. This determines the final state
-  const wrappedMethods = [getSequencerHealth, getL2NetworkStatus, getL1RollupStatus].map(_tryMethod)
+  const wrappedMethods = [
+    getSequencerHealth,
+    getL2NetworkStatus,
+    // , getL1RollupStatus // TODO
```

Comment:
Is this done on purpose or is the PR still WIP?

Reply:
Done on purpose. When Rodrigo made this adapter he left `getL1RollupStatus` empty with a TODO.
I found that confusing when reading this area of the code where it is consumed, so I marked it as such here too.

Reply:
I'm alright with the TODO here, as long as we add a gh or sc ticket to deal with it in the future.

Reply:
Only enough to make placeholder. We would have to sync up with integrations to see how they want us to check the rollup.